lib/vehicle_factory.rb

```diff
@@ -0,0 +1,21 @@
+require 'pry'
+
+class VehicleFactory
+	def initialize
+	end
+
+	def create_vehicles(registration_data)
+		registration_data.map do |info|
+			vehicle_details = {
+				vin: info[:vin_1_10], 
+				year: info[:model_year], 
+				make: info[:make], 
+				model: info[:model], 
+				engine: :ev
+			}
+														
+			Vehicle.new(vehicle_details)
+		end
+	end
+end
```

Comment:
Yup, this method looks good. 
If you have nothing in the initialize method, you don't need to include it.